models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -425,6 +422,15 @@ aeif_cond_beta_multisynapse::init_buffers_()
   B_.sys_.params = reinterpret_cast< void* >( this );
   // B_.sys_.dimension is assigned in calibrate()
   B_.I_stim_ = 0.0;
+  // set the right threshold depending on Delta_T
+  if ( P_.Delta_T > 0. )
+  {
+    B_.V_peak = P_.V_peak_;
```

Comment:
I'm almost sure this is supposed to be in calibrate since we want this check to be done every time ``Simulate`` is called, so ``V_peak`` should be in ``V_``; @heplesser, could you confirm?